template/.readme/partials/borrowed/footer.md.j2.j2

```diff
@@ -0,0 +1,56 @@
+## About The Stackable Data Platform
+
+This operator is written and maintained by [Stackable](https://www.stackable.tech) and it is part of a larger platform.
+
+![Stackable Data Platform Overview](./.readme/static/borrowed/sdp_overview.png)
+
+Stackable makes it easy to operate data applications in any Kubernetes cluster.
+
+The data platform offers many operators, new ones being added continuously. All our operators are designed and built to be easily interconnected and to be consistent to work with.
+
+Stackable GmbH is the company behind the Stackable Data Platform. Offering professional services, paid support plans and development.
+
+We love open-source!
+
+## Supported Platforms
+
+We develop and test our operators for the following cloud platforms:
+
+* Kubernetes 1.21-1.24
+* EKS on Amazon Web Services (AWS)
+* GKE on Google Cloud Platform (GCP)
+* AKS on Microsoft Azure
+* K3s
+
+We are currently working to support:
+
+* OpenShift
+
+## Other Operators
+
+These are the Operators currently part of the Stackable Data Platform:
+
+- [Apache Airflow Operator](https://github.com/stackabletech/airflow-operator)
+- [Apache Druid Operator](https://github.com/stackabletech/druid-operator)
+- [Apache HBase Operator](https://github.com/stackabletech/hbase-operator)
+- [Apache Hadoop HDFS Operator](https://github.com/stackabletech/hdfs-operator)
+- [Apache Hive Operator](https://github.com/stackabletech/hive-operator)
+- [Apache Kafka Operator](https://github.com/stackabletech/kafka-operator)
+- [Apache NiFi Operator](https://github.com/stackabletech/nifi-operator)
+- [Apache Spark Operator](https://github.com/stackabletech/spark-k8s-operator)
+- [Apache Superset Operator](https://github.com/stackabletech/superset-operator)
+- [Trino Operator](https://github.com/stackabletech/trino-operator)
+- [Apache ZooKeeper Operator](https://github.com/stackabletech/zookeeper-operator)
+- [OpenPolicyAgent Operator](https://github.com/stackabletech/opa-operator)
```

Comment:
Here im not sure, @lfrancke should this be "Stackable Operator for Apache Kafka" etc ?

Reply:
Very good catch Malte. Yes please. It sounds dumb but it's the correct term

Reply:
While we're at it - do you think this list should focus on operators for products, or include all internal operators as well? There's the commons and listener operator missing right now.

Reply:
We have the operator check list:
```
### Products
- [ ] Airflow
- [ ] Druid
- [ ] HBase
- [ ] Hadoop HDFS
- [ ] Hive
- [ ] Kafka
- [ ] NiFi
- [ ] Spark
- [ ] Superset
- [ ] Trino
- [ ] ZooKeeper

### Internal
- [ ] Commons
- [ ] Listener
- [ ] OPA
- [ ] Secret
```
We could do it the same way (with the correct public names)?

Reply:
The list is now complete, in alphabetical order, and with the right wording around the names.